profiles/Makefile.am

```diff
@@ -3,6 +3,7 @@ rawstudiodir = $(datadir)/rawstudio/profiles
 RECIPE_FILES := $(wildcard recipes/*.dcpr)
 SIMPLE_SOURCE_FILES := $(wildcard simple-src/*.xml)
 DCP_FILES := $(notdir $(SIMPLE_SOURCE_FILES:.xml=.dcp))
+DCP_FILES += $(wildcard *_Rawstudio_Advanced_Profile.dcp)
```

Comment:
Hmm, this file (being an automake setup) should actually not allow use of `$(wildcard)` because it isn't portable. However it seems to have been used before and this usage isn't making the problem any *worse*, so I'd give it a pass for now. It's worth noting that this will only work on systems with GNU Make and fall pretty flat on BSD. At some point we should refactor to avoid those problems, ,but again it doesn't have to happen now since this isn't a regression, it's just existing brokenness.

Reply:
> existing brokenness

:smiling_face_with_three_hearts: 

I'll merge right away.,